icu4c/source/i18n/listformatter.cpp

```diff
@@ -396,6 +398,7 @@ UnicodeString& ListFormatter::format_(
         int32_t &offset,
         FieldPositionHandler* handler,
         UErrorCode& errorCode) const {
+#if !UCONFIG_NO_FORMATTING
```

Comment:
There is no 'return' statement outside of the if.

Reply:
Done